10.9-libcxx/stable/main/finkinfo/devel/help2man-perl.info

```diff
@@ -43,9 +48,9 @@ CompileScript: <<
   elif [ "%type_pkg[systemperl]" = "5282" ] && [ -e "/usr/bin/perl5.28" ] ; then
     PERL="/usr/bin/arch -%m perl5.28"
   elif [ "%type_pkg[systemperl]" = "5302" ] && [ -e "/usr/bin/perl5.30" ] ; then
-    PERL="/usr/bin/arch -%m perl5.30"
```

Comment:
Should this (and again two lines later) be percent instead of dollar-sign?

Reply:
It did need it before, as %m was arm, but that is corrected to arm64 in my branch

Reply:
Gotcha